core/src/main/java/com/graphhopper/routing/util/EncodingManager.java

```diff
@@ -663,6 +670,18 @@ public String toEncodedValuesAsString() {
         return str.toString();
     }
 
+    public String toSpatialRulesAsString() {
+        StringBuilder str = new StringBuilder();
+        for (SpatialRule rule : spatialRuleLookup.getRules()) {
+            if (str.length() > 0)
+                str.append(",");
+
+            str.append(rule.getId());
+        }
+
+        return str.toString();
+    }
```

Comment:
Why do we need this? To be independent of the original spatial rule files?

Reply:
To check if the graph loaded from disk still fits the current configuration. Similar like how we treat changes to encoded values.